crates/sui-core/src/authority.rs

```diff
@@ -1226,7 +1226,7 @@ impl AuthorityState {
     }
 
     /// Check whether a shared-object certificate has already been given shared-locks.
-    async fn shared_locks_exist(&self, certificate: &CertifiedTransaction) -> SuiResult<bool> {
+    async fn transaction_locks_exist(&self, certificate: &CertifiedTransaction) -> SuiResult<bool> {
```

Comment:
The naming change could be confusing: as it doesn't check owned object locks

Reply:
good point, we should rename it to 'transaction_shared_locks_exist'

Reply:
fixed